### ISSUE TITLE
New term proposal : isotopolog_atom_per

### ISSUE THREAD

**Issue by mslarae13** [GenomicsStandardsConsortium/mixs#584]
**New term details**
For us to assess a new term request we require the following details:
```
Term name -  atom percent of isotopologue
Structured comment name - isotopolog_atom_per
Definition - [a clear and concise description of the term including ]
Expected value - float %
Value syntax - {float} %
Example - [provide and example value]
Preferred unit - [if appropriate]
Package(s) - new checklist term applicable to all packages / extensions
```

**Additional context**
Add any other context about the new term here.

**Comment by simpso91** [comment]
Term name -  atom percent of isotopologue
Structured comment name - isotopolog_atom_per
Definition - The percentage of heavy-labelled atoms out of all atoms of a given element in the isotopologue
Expected value - value
Value syntax - {float} %
Example - 98% or 98% | undefined
Preferred unit -percent

**Comment by mslarae13** [comment]
can this be multivalued? @simpso91

**Comment by simpso91** [comment]
Yes, it can be multivalued and the order should be the same as isotopologue!

**Comment by mslarae13** [comment]
@simpso91  changing this one to fraction as well? 

isotopolog_atom_fraction

Term name -  atom fraction of isotopolog
Structured comment name - isotopolog_atom_fraction
Definition - The fraction of heavy-labelled atoms out of all atoms of a given element in the isotopolog
Expected value - value (unit?)
Value syntax - {float} 
Example - ( @simpso91  any good examples?)
Preferred unit - value
Package(s) - new checklist term applicable to all packages / extensions

**Comment by simpso91** [comment]
Yes, change to fraction! The expected value is unitless and is between 0 and 1. The examples should not include '%' values anymore: 0.99 or 0.99 | 0.95 .

**Comment by mslarae13** [comment]
https://github.com/GenomicsStandardsConsortium/mixs/pull/1002
